documentation/en/source/pages/command-line/migration.rst

```diff
@@ -18,4 +18,12 @@ Sub Commands
 ``list`` sub-command
 ~~~~~~~~~~~~~~~~~~~~
 
-Displays all the migrations for the user.
\ No newline at end of file
+Displays all the migrations for the user.
+
+
+``launch`` sub-command
```

Comment:
Should remove this not to conflict with https://github.com/usharesoft/hammr/pull/357?